lib/crypto/crypto_keys.c

```diff
@@ -124,8 +126,29 @@ crypto_keys_init(void)
 {
 	uint8_t randbuf[RANDBUFLEN];
 
-	/* No keys yet. */
-	memset(&keycache, 0, sizeof(keycache));
+	/*
+	 * No keys yet.  memset() is insufficient since NULL is
```

Comment:
Line wrap when needed, not before.  Don't go past column 80; try to avoid going past column 78.  But definitely go past column 64.